contributing-guides/style-guide.md

```diff
@@ -94,6 +98,59 @@ Use backticks on the following:
 - Extensions, ex. `.dll`.
 - Commands, ex. `ls`.
 
+### Shell-specific pages
+
+If a page describes some shell (Bash for instance) then when it's possible try follow this structure:
+
+- Start an interactive shell session:
+- Start an interactive shell session without loading startup configs:
+- Execute a command:
+- Execute a script:
+- Check a script for syntax errors:
+- _any other examples_
+- Print the version:
+
+If you want to include shell builtin sample please mark it with `(builtin)` word. For example:
```

Comment:
"What in the world is a shell builtin sample" Me + most others (im guessing)

Even from reading the "For example:" I still have no idea what you are referring to.

Reply:
Now this sentence sounds like:

> If you want to include shell builtin sample please place `(builtin)` word at the end of example description such as: